data/maps/objects/Route1.asm

```diff
@@ -12,5 +12,5 @@ Route1_Object:
 
 	def_warps_to ROUTE_1
 
-	; unused
+	; unused (can be accessed via SpecialWarpIn)
```

Comment:
Why comment this?

Reply:
It's something I added while testing things. I'll remove it.

Reply:
@SatoMew I see this and probably some other comments were marked as resolved but are still the same. Did you make the changes but not commit+push them?

Reply:
Hi, yes, I wanted to commit and push once with all the changes. I'll let you know when that happens.